### ISSUE TITLE
Versioning issue

### ISSUE THREAD

**Issue by wideglide** [Cisco-Talos/binary_function_similarity#4]
https://github.com/Cisco-Talos/binary_function_similarity/blob/934e71b7c1c698dcf324e3629c8bade0714d79f8/Models/Zeek/NeuralNetwork/requirements.txt#L2

In the Zeek neural network image, python3.7 is installed and pandas 1.4.2 is listed in requirements.txt, but Pandas >1.3 requires Python3.8.  I'm assuming this will work find with `pandas==1.3.5`?

**Comment by jimmy-sonny** [comment]
This is correct and thanks for opening this issue. The Zeek neural network image works with `pandas==1.3.5`.
